k8s/overlays/cloud/cronjob.yaml.template

```diff
@@ -54,3 +54,31 @@ spec:
               args:
                 - "${KEYCLOAK_HOST}/auth/realms/europeana/zohosync"
           restartPolicy: Never
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: keycloakclient-session-count-reset-job
+spec:
+  schedule: "${KEYCLOAK_SESSIONCOUNT_RESET_SCHEDULE}"
+  suspend: ${SUSPEND_KEYCLOAK_SESSIONCOUNT_RESET}
+  startingDeadlineSeconds: 60
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      ttlSecondsAfterFinished: 86400
+      template:
+        metadata:
+          annotations:
+            fluentd/include: '${COLLECT_LOGS}'
+            fluentd/multiline: 'true'
+        spec:
+          containers:
+            - name: cron
+              image: busybox
+              imagePullPolicy: IfNotPresent
```

Comment:
we want to build it on existing old docker image ?

from what i know - If you use IfNotPresent and the image with the specified tag is already present on the worker node, Kubernetes will use that image to start the container, even if there is a newer version of the image available in the repository with the same tag.